### PR TITLE
Remove redundant alerts 

### PR DESCRIPTION
**Pull request by suiguoxin**
- `PaiServicePodNotRunning` / `PaiServicePodNotRunning` inhibits `NodeNotReady` ; 
- `PaiServiceNotUp` detects the status of `job-exporter`, `node-exporter` & `watchdog`. As services `job-exporter`, `node-exporter` are already covered by `PaiServicePodNotRunning` / `PaiServicePodNotRunning`, only `watchdog` need to be monitored in this alert.

### CODE REVIEW

File: src/prometheus/deploy/alerting/pai-services.rules [hunk 1]
```diff
@@ -34,8 +34,8 @@ groups:
         annotations:
           summary: "{{$labels.name}} in {{$labels.host_ip}} not ready detected"
 
-      - alert: PaiServiceNotUp
-        expr: up != 1
+      - alert: PaiServiceWatchDogNotUp
```
Comment (Binyang2014) [inline src/prometheus/deploy/alerting/pai-services.rules:37]: Why change this?

File: docs/manual/cluster-admin/troubleshooting.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Solutions:
 This is a kind of alert from alert manager, and is reported by watchdog service. Watchdog gets such metrics from Kubernetes API. Example metrics is like:
 
 ```
-pai_node_count{disk_pressure="false",instance="10.0.0.1:9101",job="pai_serivce_exporter",memory_pressure="false",name="10.0.0.2",out_of_disk="false",pai_service_name="watchdog",ready="true",scraped_from="watchdog-5ddd945975-kwhpr"}
+pai_node_count{disk_pressure="false",instance="10.0.0.1:9101",job="pai_serivce_exporter",memory_pressure="false",host_ip="10.0.0.2",out_of_disk="false",pai_service_name="watchdog",ready="true",scraped_from="watchdog-5ddd945975-kwhpr"}
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/troubleshooting.md:26]: Can we change to nodeName. Which is more easy to understand

Comment (suiguoxin) [reply docs/manual/cluster-admin/troubleshooting.md:26]: > Can we change to `node_name`?. Which is more easy to understand

Added `node_name` as another label. Used `node_name` instead in inhibit rules.

File: src/prometheus/deploy/alerting/pai-services.rules (hunk 1 above)
Comment (suiguoxin) [reply src/prometheus/deploy/alerting/pai-services.rules]: > Why change this?

This alert has overlap with PaiServieNotRunning|PaiServiceNotReady. Renamed to `WatchdogNotUp`.

File: docs/manual/cluster-admin/troubleshooting.md (hunk 2 above)
Comment (Binyang2014) [reply docs/manual/cluster-admin/troubleshooting.md:26]: Add `node_name` in this doc?

**Review by Binyang2014 — APPROVED** [review]

**Pull request merged**